kotti/alembic/versions/1063d7178fa_add_node_path_column.py

```diff
@@ -16,8 +16,14 @@
 
 
 def upgrade():
-    op.add_column('nodes',
-                  sa.Column('path', sa.Unicode(1000), index=True))
+    from alembic.context import get_bind
+
+    conn = get_bind()
+    if conn.engine.dialect.name == 'mysql':
+        op.add_column('nodes', sa.Column('path', sa.Unicode(1000)))
+    else:
+        op.add_column('nodes', sa.Column('path', sa.Unicode(1000), index=True))
+
```

Comment:
Can you please remove one blank line, it makes the Travis build fail due to a PEP8 error: https://travis-ci.org/Kotti/Kotti/jobs/62103570#L5529